src/main/webapp/gettingStarted.jsp

```diff
@@ -78,7 +78,8 @@
       <li>
         <b><a href="/instructorHelp.jsp#course-add-students" target="_blank" rel="noopener noreferrer">Enroll students in the course</a></b><br>
         Go to the <b>Courses</b> page and click the <button class="btn btn-default btn-xs">Enroll</button> button of the corresponding course.<br>
-        Students can be enrolled into teams (e.g. project groups) and sections (e.g. tutorial classes, lecture groups) to facilitate giving feedback in and among these smaller groups.
+        Students can be enrolled into teams (e.g. project groups) and sections (e.g. tutorial classes, lecture groups) to facilitate giving feedback in and among these smaller groups.<br>
+        TEAMMATES will <b>not</b> automatically notify students that they have been enrolled. However, you may choose to <a href="/instructorHelp.jsp#student-email" target="_blank" rel="noopener noreferrer">email students</a> if you would like them to fill up their <a href="/instructorHelp.jsp#student-view-profile" target="_blank" rel="noopener noreferrer">student profiles</a>.
```

Comment:
I believe the correct way is to click `remind student to join`? You can move the `In addition, if you would like students to access TEAMMATES sooner (e.g. if you would like them to fill in their profile page in advance), xxx` in section 3 to here and modify it so there won't be duplication.